src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -769,12 +758,10 @@ private static uint HandleEvent(State state, ref StreamEvent evt)
             {
                 if (NetEventSource.Log.IsEnabled())
                 {
-                    NetEventSource.Error(state, $"[Stream#{state.GetHashCode()}] Exception occurred during handling {(QUIC_STREAM_EVENT_TYPE)evt.Type} event: {ex}");
+                    NetEventSource.Error(state, $"[Stream#{state.GetHashCode()}] Exception occurred during handling {evt.Type} event: {ex}");
                 }
 
-                // This is getting hit currently
-                // See https://github.com/dotnet/runtime/issues/55302
-                //Debug.Fail($"[Stream#{state.GetHashCode()}] Exception occurred during handling {(QUIC_STREAM_EVENT_TYPE)evt.Type} event: {ex}");
+                Debug.Fail($"[Stream#{state.GetHashCode()}] Exception occurred during handling {evt.Type} event: {ex}");
```

Comment:
I think we should find better way how to propagate exceptions to the caller.  This is going to be hard IMHO to debug in CI and does nothing to diagnose filed issues.

Reply:
I agree, it's a bit of a pain to deal with this.

That said, it's not obvious to me how to propagate this to the caller and/or make it easier to debug. Suggestions welcome.

Hopefully this is not a common occurrence -- an exception during a callback means something is deeply, deeply wrong.